vestapol/typing/resources.py

```diff
@@ -0,0 +1,6 @@
+from vestapol.web_resources.csv_resource import CSVResource
+from vestapol.web_resources.github_resource import GitHubResource
+from vestapol.web_resources.json_resource import JSONResource
+from typing import Union
+
+ResourceTypes = Union[CSVResource, GitHubResource, JSONResource]
```

Comment:
I might suggest moving this file to `vestapol/web_resources/__init__.py`, the idea being that, if we were to add a new resource, all changes would be encapsulated within `vestapol/web_resources/`. 

We'd essentially be restoring the `__init__.py` files [here](https://github.com/phillymedia/vestapol/pull/9/files#diff-3ed08c9df8c5cb9731c8cff880eef81e75073b6d0ce12f38dc0634030747ef29) and adding lines 4-6 above.